prober/derp.go

```diff
@@ -200,7 +211,11 @@ func (d *derpProber) probeMapFn(ctx context.Context) error {
 					n := fmt.Sprintf("derp/%s/%s/%s/bw", region.RegionCode, server.Name, to.Name)
 					wantProbes[n] = true
 					if d.probes[n] == nil {
-						log.Printf("adding DERP bandwidth probe for %s->%s (%s) %v bytes every %v", server.Name, to.Name, region.RegionName, d.bwProbeSize, d.bwInterval)
+						tunString := ""
+						if d.bwTUNAddress != "" {
```

Comment:
We probably want to validate the flag before spinning up probes in order to exit the process if the input is invalid